app/views/posts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to "back to post",url_for(:back) %>
+<%= form_with model: @post, local:true, data:{turbo: false} do |form| %>
+  <p>
+    <%= form.label :title %>
+    <%= form.text_field :title %>
+  </p>
+
+  <p>
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </p>
+  <p>
+    <%= form.submit %>
+  </p>
```

Comment:
- [ ] As required by the previous reviewer, please ensure that when the submit button is clicked in the process of creating a post, the post is actually created. At the moment, the post is never created 👍

Reply:
Thanks fixed